pkg/apis/scorecard/v1alpha2/formatter.go

```diff
@@ -46,27 +46,37 @@ func (s ScorecardOutput) MarshalText() (string, error) {
 			sb.WriteString(fmt.Sprintf("%s:\n", result.Labels["suite"]))
 			currentSuite = result.Labels["suite"]
 		}
-		sb.WriteString(fmt.Sprintf("\t%-35s: ", result.Name))
+		sb.WriteString(fmt.Sprintf("\t%-35s ", result.Name))
 
 		if result.State == PassState {
 			sb.WriteString(fmt.Sprintf(passColor, PassState))
-		} else {
+		} else if result.State == FailState {
 			sb.WriteString(fmt.Sprintf(failColor, FailState))
+		} else {
+			sb.WriteString(fmt.Sprintf("\n"))
 		}
-	}
 
-	for _, result := range s.Results {
-		for _, suggestion := range result.Suggestions {
+		sb.WriteString("\tLabels: \n")
+		for labelKey, labelValue := range result.Labels {
+			sb.WriteString(fmt.Sprintf("\t\t%s:%s\n", labelKey, labelValue))
```

Comment:
Should we put labels and values in quotes in case they have spaces in them? I think for our own labels this should never be a problem, but if/when we add support back in for external plugins with v1alpha2, we'll have less control.
```suggestion
			sb.WriteString(fmt.Sprintf("\t\t%q: %q\n", labelKey, labelValue))
```

Reply:
I'll add this, it renders pretty good I think.